<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::ReportView</class>
 <widget class="QDialog" name="QGBA::ReportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Bug Report</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,2,7">
   <item row="1" column="1" rowspan="3">
    <widget class="QListWidget" name="fileList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QPlainTextEdit" name="fileView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditorInteraction</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To file a bug report, please first generate a report file to attach to the bug report you're about to file. It is recommended that you include the save files, as these often help with debugging issues. This will collect some information about the version of {projectName} you're running, your configuration, your computer, and the game you currently have open (if any). Once this collection is completed you can review all of the information gathered below and save it to a zip file. The collection will automatically attempt to redact any personal information, such as your username if it's in any of the paths gathered, but just in case you can edit it afterwards. After you have generated and saved it, please click the button below or go to &lt;a href=&quot;https://mgba.io/i/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;mgba.io/i&lt;/span&gt;&lt;/a&gt; to file the bug report on GitHub. Make sure to attach the report you generated!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate report</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>../../../../../../</normaloff>../../../../../../</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>../../../../../../</normaloff>../../../../../../</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openList">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Open issue list in browser</string>
       </property>
       <property name="icon">
        <iconset theme="document-send">
         <normaloff>../../../../../../</normaloff>../../../../../../</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="includeSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Include save file</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="includeState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Create and include savestate</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openList</sender>
   <signal>clicked()</signal>
   <receiver>QGBA::ReportView</receiver>
   <slot>openBugReportPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generate</sender>
   <signal>clicked()</signal>
   <receiver>QGBA::ReportView</receiver>
   <slot>generateReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>clicked()</signal>
   <receiver>QGBA::ReportView</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generateReport()</slot>
  <slot>save()</slot>
  <slot>openBugReportPage()</slot>
 </slots>
</ui>
